CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/resources/delimited-importer.css

```diff
@@ -0,0 +1 @@
+/* Delimited Importer */
```

Comment:
This file is empty so can it be removed?

Reply:
Yep I think so.